src/azure-cli/azure/cli/command_modules/acr/check_health.py

```diff
@@ -204,9 +205,14 @@ def _get_registry_status(login_server, ignore_errors):
 
     import requests
     from azure.cli.core.util import should_disable_connection_verify
-    challenge = requests.get('https://' + login_server + '/v2/', verify=(not should_disable_connection_verify()))
 
-    if challenge.status_code == 403:
+    ssl_error_occured = False
+    try:
+        challenge = requests.get('https://' + login_server + '/v2/', verify=(not should_disable_connection_verify()))
+    except SSLError:
+        ssl_error_occured = True
+
+    if ssl_error_occured or challenge.status_code == 403:
```

Comment:
You might want to create a specific error to SSL error. The CONNECTIVITY_FORBIDDEN_ERROR is specific to 403 status.
Probably something like "Are you running a proxy?" or "Please restart your machine and retry again"

Reply:
@sajayantony Can we add this message? "Are you running a proxy? Please restart your machine and retry again."

Reply:
This should point to a deep link on the error reference  like this - https://aka.ms/acr/errors#connectivity_forbidden_error
In the reference docs you can call out the details of the fact that this has been observed when the client is behind a proxy. Also maybe we can recommend they setup the environment variable that disables SSL verification on the CLI.

Reply:
@sajayantony do you think that all errors should also have the deep link?

Reply:
Yes that would be good. 
Also we have an item to suggest running check health for other commands like failed login.